coding-concepts.js

```diff
@@ -8,39 +8,39 @@
 // --------------------1) What will this log?
 
 var cohort = "Bravo 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ['B','r','a','v','o',' ','2','0','2','2']
+// b) Verify and explain: .split() splits strings into an array of substrings and outputs the array
 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 'Hello, LEARN Student!'
+// b) Verify and explain: undefined - name is undefined? Improper syntax?
```

Comment:
It's a bit of a tricky one. Having a function return undefined is a very common error. When this happens it is a good idea to first look for a return.